Core/MIPS/ARM/ArmAsm.cpp

```diff
@@ -15,6 +15,8 @@
 // Official git repository and contact information can be found at
 // https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
 
+#include "ppsspp_config.h"
+#if PPSSPP_ARCH(ARM)
```

Comment:
Yeah, but I think I'm fine with having them until I start go digging in the armjits again.